config-provisioning/src/main/java/com/yahoo/config/provision/ClusterSpec.java

```diff
@@ -79,10 +77,8 @@ public Optional<Id> combinedId() {
         return combinedId;
     }
 
-    /**
-     * Returns whether the physical hosts running the nodes of this application can
-     * also run nodes of other applications. Using exclusive nodes for containers increases security and cost.
-     */
+
+    /** Returns whether the physical hosts must be provisioned specifically for this application. */
```

Comment:
The previous documentation also covered zones with host sharing (Yahoo)

Reply:
Let me restore the previous docstring.